### ISSUE TITLE
Update VM

### ISSUE THREAD

**Issue by jeamick** [asseth/assethbox#8]
When I want to use truffle Box, You need **webpack-dev server** and **webpack** .

Is it possible to have the [**truffle box**](https://truffle-box.github.io/) integrate with the next update VM ?

I think we need  
     - **Webpack** : `$ npm install webpack -g `
     - **Webpack-dev-server** : `npm install webpack-dev-server -g`
     - **React**  : `npm install -g create-react-app`
     - **Truffle box** : there are 3  with react , redux and Uport

By the same time, we can update/install some software like : 
     - **Truffle**: Current version  v.3.2.4 to v.3.4.9
     - ** web3-provider-engine** : In order to use Infura : `npm install ethereumjs-wallet bip39 web3-provider-engine web3`
     - Others softwares if needed

**Comment by kurthalex** [comment]
yes jeamick I hope next release will be soon, sorry for the delay :-)
